### DISCUSSION TITLE
How to disable Camera

### DISCUSSION THREAD

**Discussion by yukioshima** [Sixze/ALS-Refactored#280] [Q&A] [answered]
Is there any way to disable the ALS camera and replace it with another camera?
Currently, the camera cannot be removed and replaced.

**Comment by ameaninglessname** [comment]
You need to remove it from `AAlsCharacterExample::AAlsCharacterExample`and related code if you are using `AAlsCharacterExample` as parent class

**Comment by assmikDuDuDuJ** [comment, marked as answer]
Keep it simple, just select the camera in blueprint and turn off "Auto Activate"
This way you can switch between the different cameras by just turning on the one you want and turning the others off. 
or is that a bad thing to do?

**Reply by yukioshima** [reply]
Thank you for your help! We were looking for exactly that feature.
